pkgs/top-level/python-packages.nix

```diff
@@ -26937,4 +26937,113 @@ in modules // {
       license = licenses.wtfpl;
     };
   };
+
+  uptime = buildPythonPackage rec {
+    name = "uptime-${version}";
+    version = "3.0.1";
+
+    src = pkgs.fetchFromGitHub {
+      owner = "Cairnarvon";
+      repo = "uptime";
+      rev = "1ddfd06bb300c00e6dc4bd2a9ddf9bf1aa27b1bb";
+      sha256 = "148rqj0asw65c3z4whnqphfmkhlm181p296b7pxc1x79n97gn1pf";
+    };
+
+    meta = {
+      description = "Cross-platform uptime library for the Python";
+      homepage = "https://github.com/Cairnarvon/uptime";
+    };
+  };
+
+  pytestHttpbin = buildPythonPackage rec {
+    name = "pytest-httpbin-${version}";
+    version = "0.2.0";
+
+    src = pkgs.fetchFromGitHub {
+      owner = "kevin1024";
+      repo = "pytest-httpbin";
+      rev = "ae314b2a18eecac2a9d3b2034bfede9805ce7907";
+      sha256 = "0j5rcqlh59sqcv6csnk0i73xi5p43h32ynnz550q8pv9q68lzqa2";
+    };
+
+    buildInputs = with self; [
+      pytest
+    ];
+
+    propagatedBuildInputs = with self; [
+      flask
+      decorator
+      httpbin
+      six
+    ];
+
+    meta = {
+      description = "Easily test your HTTP library against a local copy of httpbin.org";
+      homepage = "https://github.com/kevin1024/pytest-httpbin";
+    };
+  };
+
+  vcrpy = buildPythonPackage rec {
+    name = "vcrpy-${version}";
+    version = "1.7.4";
+
+    src = pkgs.fetchFromGitHub {
+      owner = "kevin1024";
+      repo = "vcrpy";
+      rev = "v${version}";
+      sha256 = "1inm22xlnlb4xxcrrmrm3j73kffksb2838ypgykjbsjid8xvqdsd";
+    };
+
+    # https://github.com/kevin1024/vcrpy/commit/4e36997e1af248a96b1424b6975f5022baa9352f
+    # test_urllib2 fails with SSL_VERIFICATION_ERROR. Above commit (unreleased)
+    # sets the correct cacert path
+    doCheck = false;
+
+    buildInputs = with self; [
+      pytest
+      mock
+      pytestHttpbin
+      pytest-localserver
+    ];
+
+    propagatedBuildInputs = with self; [
+      pyyaml
+      six
+      wrapt
+      contextlib2
+      mock
+    ];
+
+    meta = {
+      description = "Automatically mock your HTTP interactions to simplify and speed up testing";
+      homepage = "https://github.com/kevin1024/vcrpy";
+    };
+  };
+
+  pyvmomi = buildPythonPackage rec {
+    name = "pyvmomi-${version}";
+    version = "6.0.0";
+
+    src = pkgs.fetchFromGitHub {
+      owner = "vmware";
+      repo = "pyvmomi";
+      rev = "51a54c0ca3c07aa7c3d4314d962efbce2d5b81f4";
+      sha256 = "1qxrbrrfy82dl6yqg6mffwvyzv0xwxfqpp61p1mf7ngy0cxsac50";
+    };
```

Comment:
and https://pypi.python.org/pypi/pyvmomi/6.0.0
or 
https://pypi.python.org/pypi/pyvmomi/6.0.0.2016.4